internal/internal.go

```diff
@@ -677,19 +684,31 @@ func uuid4() (string, error) {
 	return fmt.Sprintf("%x-%x-%x-%x-%x", bs[0:4], bs[4:6], bs[6:8], bs[8:10], bs[10:]), nil
 }
 
-func getRevision(ctx context.Context, store storage.Store, txn storage.Transaction) (string, error) {
-	value, err := store.Read(ctx, txn, revisionPath)
-	if err != nil {
-		if storage.IsNotFound(err) {
-			return "", nil
+func getRevision(ctx context.Context, store storage.Store, txn storage.Transaction, result *evalResult) error {
+	revisions := map[string]string{}
+
+	names, err := bundle.ReadBundleNamesFromStore(ctx, store, txn)
+	if err != nil && !storage.IsNotFound(err) {
+		return err
+	}
```

Comment:
If `storage.IsNotFound(err)` is true, is it safe to use `names`? (It probably is, just wanting to be sure)

Reply:
I think it is.